client/my-sites/domain-tip/index.jsx

```diff
@@ -22,14 +23,15 @@ import UpgradeNudge from 'my-sites/upgrade-nudge';
 import { FEATURE_CUSTOM_DOMAIN } from 'lib/plans/constants';
 import { isFreePlan } from 'lib/products-values';
 
-function getQueryObject( site, siteSlug ) {
+function getQueryObject( site, siteSlug, vendor ) {
 	if ( ! site || ! siteSlug ) {
 		return null;
 	}
 	return {
 		query: siteSlug.split( '.' )[ 0 ],
 		quantity: 1,
-		vendor: 'domainsbot',
+		vendor,
+		recommendation_context: ( site.name || '' ).replace( ' ', ',' ).toLocaleLowerCase(),
```

Comment:
Looks like this needs to be passed explicitly to `QueryDomainsSuggestions`? Otherwise, it's useless - unless I missed the purpose of it 🤔

Reply:
Ah, good catch... I'll fix this.